Assets/Scripts/Level/Level.cs

```diff
@@ -118,75 +117,159 @@ private void Start()
+            
+            // MIDBOSS
+            les.Add(new EnemySpawn{enemy = midboss, spawnPosition = new Vector3(0, 4f, 0), spawnTime = 105f, waitUntilDead = true});
+            
+            /*
+             * 2:10 - 2:40
+             */
+            les.CurvedWave(5, 135f, roid1, new Vector3(-5, 3, 0), EntrySide.Left);
+            
+            les.CurvedWave(5, 140f, roid2, new Vector3(5, 3.5f, 0), EntrySide.Right);
+            
+            les.TopWave(5, 145f, roid3, new Vector3(-3, 6, 0), EntrySide.Left);
+            
+            les.CurvedWave(5, 150f, roid2, new Vector3(5, 4f, 0), EntrySide.Right);
+            
+            les.TopWave(5, 155f, roid3, new Vector3(3, 6, 0), EntrySide.Right);
+            
+            les.CurvedWave(5, 160f, roid1, new Vector3(-5, 4, 0), EntrySide.Left);
+            
+            les.TopWave(5, 165f, roid3, new Vector3(3, 6, 0), EntrySide.Right);
+            
+            /*
+             * 2:40 - 3:00
+             */
+            les.Add(new EnemySpawn{enemy = bomber1, spawnPosition = new Vector3(3, 6, 0), spawnTime = 170});
+            les.CurvedWave(5, 171, roid1, new Vector3(-5,3,0), EntrySide.Left);
+            
+            les.Add(new EnemySpawn{enemy = bomber1, spawnPosition = new Vector3(-3, 6, 0), spawnTime = 176});
+            les.CurvedWave(5, 177, roid2, new Vector3(5,3,0), EntrySide.Right);
+            
+            les.Add(new EnemySpawn{enemy = bomber1, spawnPosition = new Vector3(0, 6, 0), spawnTime = 182});
+            les.CurvedWave(5, 183, roid2, new Vector3(5,3,0), EntrySide.Right);
+            les.CurvedWave(5, 183, roid1, new Vector3(-5,3,0), EntrySide.Left);
+            
+            les.TopRow(5, 189, roid3, new Vector3(-4, 5, 0), EntrySide.Left);
+            
+            les.TopRow(5, 195, roid3, new Vector3(4, 5, 0), EntrySide.Right);
+            
+            // BOSS
+            les.Add(new EnemySpawn{enemy = boss, spawnPosition = new Vector3(0, 4f, 0), spawnTime = 205f, waitUntilDead = true});
+            
             // Sort list by time and convert to queue
             spawnQueue = les.GetSpawnQueue();
+            playSongWhenSpawnEnemyCount.Add(spawnQueue.Count);
         }
 
         void Update()
         {
+            // Do nothing if player has won or lost
+            if (lost || won) return;
+            
             // Update time elapsed, accounting for sliptime
             timeElapsed += Time.deltaTime * sliptime.slipTimeCoefficient;
-            // Handle win/lose conditions before dealing with new enemies
-            if (lost || won) return;
-            if (!lost && playerObject.IsDead())
+            
+            // Handle win/lose conditions before spawning enemies
+            if (!lost && playerLiving.IsDead())
             {
                 lost = true;
                 scoreScreenManager.Lose();
             }
-            else if(!won && bossObject.IsDead())
+            else if(bossLiving != null && !won)
```

Comment:
I don't like doing the check this way because comparing to null is expensive. If someone has a better way of doing this check, lmk and I'll do that instead. This does work, but it isn't optimal.